cpp/inc/bond/ext/grpc/unary_call.h

```diff
@@ -149,37 +159,58 @@ class unary_call final
     }
 
     /// @brief Get the server context for this call.
+    ///
+    /// @note May not be called on this instance if it has been moved-from.
```

Comment:
Put a single comment about the effects of moving on move copy/assign operator only?